.vale/styles/OpenShiftAsciiDoc/SuggestAttr.yml

```
@@ -0,0 +1,80 @@
+---
+extends: substitution
+ignorecase: true
+level: suggestion
+message: "When possible, use attributes for items such as the official name of the product. For a list of available attributes, see https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc"
```

Use the "link" field for links. This adds a clickable link to the IDE pop up. 

Also can you make the error message more direct?

```suggestion
message: "Don't use a plain text product term where an AsciiDoc attribute is available for the term."
link: https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc
```